SPECS/python2/python2.spec

```diff
@@ -31,6 +31,8 @@ Patch10:        CVE-2007-4559.nopatch
 Patch11:        CVE-2019-18348.nopatch
 # CVE-2020-27619 patch backported from 3.6
 Patch12:        CVE-2020-27619.patch
+# CVE-2021-23336 patch backported from 3.6 courtesy of Gentoo
```

Comment:
Suggestion: also add a link to their backport for the sake of documenting our sources, even though we have the copy in the .patch file.

Reply:
Added, thanks!